### ISSUE TITLE
Unimplemented VTST.8 instruction

### ISSUE THREAD

**Issue by EmulationFanatic** [Ryujinx/Ryujinx#1338]
Requires the VTST.8 instruction.

```
The instruction at 0x000000003afc0c68 (opcode 0xf24ac8f4) is undefined!
```
**Description:**
```
vtst.8 q14, q13, q10
```

## Required by: 
https://github.com/Ryujinx/Ryujinx-Games-List/issues/1350

**Comment by piyachetk** [comment]
Isn't this issue already resolved by #1192 a long time ago? The added opcode matches the one in the description.

**Comment by gdkchan** [comment]
Right, thanks for leting us know.
